apertium-eng.eng.rlx

```diff
@@ -328,7 +333,13 @@ SELECT (adv itg) IF (-1 BOS) ;
 SELECT V IF (-1 (prn pers)) ;
 SELECT SUB:1 Be IF (1 Ing) (0/* Have) ;
 SELECT Ger IF (-1/* Be) (1 To) ;
-
+SELECT Adv IF (0 Adj) (-1 Vblex) ;
+SELECT Pr IF (0 Cnjadv) (1 Det) ;
```

Comment:
This one doesn't work either, I think.  You could have "He didn't know, for the cat didn't say." where it would give you the wrong reading.

Reply:
Are there any other words that are ambiguous between Cnjadv and Pr?

Reply:
I've added another rule so that this works.